src/olympia/devhub/forms.py

```diff
@@ -762,6 +762,7 @@ class DistributionChoiceForm(happyforms.Form):
         choices=(
             ('listed', mark_safe_lazy(LISTED_LABEL)),
             ('unlisted', mark_safe_lazy(UNLISTED_LABEL))),
+        initial= 'listed',
```

Comment:
Please remove extra space before `'listed'`

Reply:
@diox I have amended the commit